### DISCUSSION TITLE
통계 기능에 대한 의견을 받고 싶습니다

### DISCUSSION THREAD

**Discussion by wonderlandpark** [koreanbots/koreanbots#138] [일반] [answered]
# 통계 기능

## 설명

봇 페이지의 상호작용 정보 통계들을 봇 개발자들에게 제공.

## 목적

서버, 하트 수 변경 추이 등과 같은 봇 상승세를 예측, 확인하는데 유용한 정보들을 봇 개발자가 따로 구현할 필요없이 사이트에서 제공.

## 의견

봇 통계 기능에서 제공하면 유용할거 같은 정보들을 제안해주시거나 해당 기능에 관련해 어떠한 의견이라도 좋습니다.

## 질문

[공식 디스코드 서버](https://koreanbots.dev/discord)나 깃허브내 Discussion 기능을 이용해주세요.

**Comment by samsunghappytree123** [comment]
봇 통계 기능은 좋은데 그럼 통계는 어느 페이지에서, 어떻게 보여주실껀가요?

**Reply by ghost** [reply]
통계 정보를 긁어올 수 있게 좀 정리? 해주셔서 봇이 긁어올 수 있게 만들어주셔도 괜찮을 것 같네요

**Reply by wonderlandpark** [reply]
관리패널에 표시할 생각입니다

**Reply by wonderlandpark** [reply]
> 통계 정보를 긁어올 수 있게 좀 정리? 해주셔서 봇이 긁어올 수 있게 만들어주셔도 괜찮을 것 같네요

@jjun4341 답글이 아닌 새로운 코맨트로 남겨주세요

**Comment by samsunghappytree123** [comment]
카트라이더 tmi처럼 통계를 공개적으로 오픈하는건 떨까요?

**Reply by wonderlandpark** [reply]
봇에 관한 통계는 민감할 수 있는 정보이기에 공개적으로 오픈할 의향은 없습니다.

**Reply by wonderlandpark** [reply]
봇이 자기 자신의 통계를 API에서 확인할 수는 있을거 같습니다.

**Reply by samsunghappytree123** [reply]
> 봇이 자기 자신의 통계를 API에서 확인할 수는 있을거 같습니다.

그럼 api로 확인할 수 있게 해주시면 좋을 것 같고, `관리 패널` 에서 하려면 조금 불편한 면이 있어 질문드려봤습니다!

**Reply by mome0320** [reply]
그러면 그 제작자가 공개 범위를 설정하는 것도 좋을 거 같은데.

**Reply by wonderlandpark** [reply]
의견 감사합니다.

**Comment by paring-chan** [comment]
서버수 변동되는것도 표시되면 좋겠네요

**Reply by wonderlandpark** [reply]
의견 감사합니다

**Comment by wonderlandpark** [comment, marked as answer]
의견 남겨주셔서 감사합니다. 추후 v2에서 만나보실 수 있도록 노력하겠습니다. 감사합니다!!

**Comment by kimjunseong04** [comment]
서버수 변동수치와 좋아요 변동 등등을 확인하면 좋을것같습니다!

부가적으로는 봇이 언제 오프라인이었는지도 뜨면 좋을것같습니다!
